php/class-customize-concurrency.php

```diff
@@ -14,4 +14,204 @@
  */
 class Customize_Concurrency {
 
-}
+	/**
+	 * Post type that stores settings changes until session ends.
+	 *
+	 * Limited by WP to 20 characters
+	 *
+	 * @var string $POST_TYPE
+	 */
+	const POST_TYPE = 'custom_saved_setting';
```

Comment:
Man, 20 chars is a rough limit!

Some alternatives that I think would be better: `customize_save_value` or `customize_setting`